load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -1479,3 +1570,36 @@ func (s *relaysSuite) populateWithKnownGateways() (gateways []*accountInfo) {
 
 	return gateways
 }
+
+// WaitAll waits for all the provided functions to complete.
+// It is used to wait for multiple goroutines to complete before proceeding.
+func (s *relaysSuite) WaitAll(waitFuncs ...func()) {
+	wg := sync.WaitGroup{}
+	wg.Add(len(waitFuncs))
+
+	for _, f := range waitFuncs {
+		go func(f func()) {
+			f()
+			wg.Done()
+		}(f)
+	}
+
+	wg.Wait()
+}
+
+// abciEventsToTypedEvents converts the abci events to typed events.
+func abciEventsToTypedEvents(ctx context.Context, abciEventObs observable.Observable[[]types.Event]) observable.Observable[[]proto.Message] {
```

Comment:
OPTIONAL: What do you think of moving this into `poktroll/testutil/events/filter.go`?